web_modules/layouts/HomePage/index.js

```diff
@@ -39,11 +39,12 @@ export default class HomePage extends Component {
       { property: "og:title", content: head.title },
       { property: "og:type", content: "article" },
       { property: "og:url", content: this.props.__url },
-      // { property: "og:description", content: pageDescription(body) },
+      { property: "og:description", content: head.description },
+
       { name: "twitter:card", content: "summary" },
       { name: "twitter:title", content: head.title },
       { name: "twitter:creator", content: `@${ pkg.twitter }` },
-      // { name: "twitter:description", content: pageDescription(body) },
```

Comment:
@thangngoc89 In #199 @marcustisater advised me to use https://github.com/postcss/postcss.org/blob/master/web_modules/layouts/Page/index.js for reference, which omits a twitter:description for a normal meta description.

However, now that I review https://dev.twitter.com/cards/types/summary, I see that twitter:description is considered required for Summary Cards. I'll add it.